service/src/main/java/com/softserve/borda/services/UserService.java

```diff
@@ -12,15 +12,15 @@
     User createOrUpdate(User user);
     void deleteUserById(Long id);
 
-    List<Board> getBoardsByUserId(Long id);
+    List<Board> getBoardsByUser(User user);
```

Comment:
I think the previous format was better.

Reply:
Why info `UserService` returns `Board` entity?

Reply:
Yes, returned to previous version. Fixed in ReworkService branch/ticket